### PR TITLE
GEOPY-571: Merge shared_utils.utils.rotate_xy with utils.models.rotate_xyz

### PR DESCRIPTION
**Pull request by jamiefb**
**GEOPY-571 - Merge shared_utility.utils.rotate_xy with utils.models.rotate_xyz**

### CODE REVIEW

File: geoapps/shared_utils/utils.py [hunk 1]
```diff
@@ -328,26 +328,50 @@ def filter_xy(
...
     """
-    Perform a counterclockwise rotation on the XY plane about a center point.
+    Rotate scatter points in column format around a center location.
 
-    :param xyz: shape(*, 3) Input coordinates
-    :param center: len(2) Coordinates for the center of rotation.
-    :param  angle: Angle of rotation in degree
+    :param xyz: shape(*, 2) or shape(*, 3) Input coordinates.
+    :param center: len(2) or len(3) Coordinates for the center of rotation.
+    :param theta: Angle of rotation around z-axis in degrees.
```
Comment (benk-mira) [inline geoapps/shared_utils/utils.py:337]: Should state the direction here.  Are they still counterclockwise?

**Review by benk-mira — CHANGES_REQUESTED** [review]
Great - thanks for this!  Let's just add to the docstrings what direction the rotations are for.

File: geoapps/shared_utils/utils.py [hunk 2]
```diff
@@ -328,26 +328,50 @@ def filter_xy(
     return mask
 
 
-def rotate_xy(xyz: np.ndarray, center: list, angle: float):
+def rotate_xyz(xyz: np.ndarray, center: list, theta: float, phi: float = 0.0):
     """
-    Perform a counterclockwise rotation on the XY plane about a center point.
+    Perform a counterclockwise rotation of scatter points around the z-axis, then x-axis, about a center point.
```
Comment (benk-mira) [inline geoapps/shared_utils/utils.py:333]: Perfect, thanks

**Review by benk-mira — DISMISSED** [review]
That should do it!

**Review by domfournier — APPROVED** [review]

**Pull request merged**